tests/test_wind_energy.py

```diff
@@ -588,47 +588,6 @@ def test_val_no_grid_land_pts(self):
             os.path.join(args['workspace_dir'], 'output', vector_path),
             os.path.join(REGRESSION_DATA, 'priceval', vector_path))
 
-    def test_field_error_missing_bio_param(self):

```

Comment:
This test was testing that `wind_energy.execute()` raises an error when a csv is missing a column. Removed those checks because it seems to duplicate the work that `validate` should already be doing.